wake-word-voice-assistant/m5stack-atom-echo.yaml

```diff
@@ -1 +1,351 @@
+          max_brightness: 100%
+
+script:
+  - id: reset_led
+    then:
+      - if:
+          condition:
+            - switch.is_on: use_micro_wake_word
+            - switch.is_on: use_listen_light
+          then:
+            - light.turn_on:
+                id: led
+                red: 100%
+                green: 89%
+                blue: 71%
+                brightness: 60%
+                effect: none
+          else:
+           - if:
+              condition:
+                - switch.is_off: use_micro_wake_word
+                - switch.is_on: use_listen_light
+              then:
+                - light.turn_on:
+                    id: led
+                    red: 0%
+                    green: 100%
+                    blue: 100%
+                    brightness: 60%
+                    effect: none
+              else:
+               - light.turn_off: led
+
+switch:
+  - platform: template
+    name: Use micro wake word
+    id: use_micro_wake_word
+    optimistic: true
+    restore_mode: RESTORE_DEFAULT_ON
+    entity_category: config
+    on_turn_on:
+      - voice_assistant.stop:
+      - lambda: id(va).set_use_wake_word(false);
+      - delay: 1s
+      - micro_wake_word.start:
+      - script.execute: reset_led
+    on_turn_off:
+      - micro_wake_word.stop:
+      - lambda: id(va).set_use_wake_word(true);
+      - delay: 1s
+      - voice_assistant.start_continuous:
+      - script.execute: reset_led
+  - platform: template
+    name: Use listen light
+    id: use_listen_light
+    optimistic: true
+    restore_mode: RESTORE_DEFAULT_ON
+    entity_category: config
+    on_turn_on:
+      - script.execute: reset_led
+    on_turn_off:
+      - script.execute: reset_led
+  - platform: template
+    id: timer_ringing
+    optimistic: true
+    internal: true
+    restore_mode: ALWAYS_OFF
+    on_turn_on:
+      - delay: 15min
+      - switch.turn_off: timer_ringing
+
+external_components:
+  - source: github://pr#5230
+    components:
+      - esp_adf
+    refresh: 0s
+  - source: github://jesserockz/esphome-components
+    components: [file]
+    refresh: 0s
+  - source:
+      type: git
+      url: https://github.com/kahrendt/esphome
+      ref:  mww-v2-external-library
+    refresh: 0s
+    components: [ micro_wake_word ] 
+
+esp_adf:
+
+file:
+  - id: timer_finished_wave_file
+    file: https://github.com/esphome/firmware/raw/main/voice-assistant/sounds/timer_finished.wav 
+
+micro_wake_word:
+  on_wake_word_detected:
+    - voice_assistant.start: 
+        wake_word: !lambda return wake_word; 
+  vad:
+    model: https://github.com/kahrendt/microWakeWord/releases/download/v2.1_models/vad.json
+  models:
+    - model: https://github.com/kahrendt/microWakeWord/releases/download/v2.1_models/okay_nabu.json
+#    - model: https://github.com/kahrendt/microWakeWord/releases/download/v2.1_models/hey_jarvis.json
+#    - model: https://github.com/kahrendt/microWakeWord/releases/download/v2.1_models/hey_mycroft.json
```

Comment:
I would recommend to use:

```suggestion
  vad:
  models:
    - model: okay_nabu
#    - model: hey_jarvis
#    - model: hey_mycroft
```

> It will load the default VAD model without specifying things further. If there is a different VAD model available, then you can specify it using your syntax by pointing it to a local file or a file on github. No other models are available at the moment.

https://discord.com/channels/429907082951524364/1209968717904216084/1263151014094307378

Reply:
Thank you